util/src/main/java/tech/pegasys/teku/util/config/Constants.java

```diff
@@ -109,6 +109,8 @@
   public static final int STORAGE_QUERY_CHANNEL_PARALLELISM = 10; // # threads
   public static final int PROTOARRAY_FORKCHOICE_PRUNE_THRESHOLD = 256;
 
+  public static final Duration TTD_MONITOR_SERVICE_POLL_INTERVAL = Duration.ofSeconds(3);
```

Comment:
I'd say this should be more like 15 seconds at a minimum.  Getting it from `spec.getSecondsPerEth1Block` would be even better (just using the value from genesis would be fine - it's not going to be changed prior to the merge now).

Reply:
I was incline to poll more frequently than expected block time. So you're saying is better to react slower but with a block that is less likely to be reorged?

Reply:
I think we want to avoid making too many requests to the execution layer. The reality is that we need this information once - eth1 blocks come about every 14 seconds and block proposals that need this info are at most every 12 seconds but most likely separated by hours.  We just don't need to pick up the TTD block within 3 seconds.